modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/DefaultPage.js

```diff
@@ -37,16 +37,14 @@ describe('DefaultPage', () => {
 		expect(getByText('Page title')).toBeInTheDocument();
 	});
 
-	it('shows submit again button when Limit to one submission per user is disabled', () => {
-		const {getByText} = render(<DefaultPage />);
+	it('shows submit again button when the show submit again flag is disabled', () => {
```

Comment:
@rodrigopaulino shouldn’t be '`shows submit again button when the show submit again flag is enabled`'?

Reply:
Oh boy. 🤭 It should. Thanks again, @carolmariaabb